sdk/core/azure-core/src/main/java/com/azure/core/http/policy/HttpLogOptions.java

```diff
@@ -168,7 +171,9 @@ public String getApplicationId() {
      *
      * @param applicationId The user specified application id.
      * @return The updated HttpLogOptions object.
+     * @deprecated deprecated in favor of {@link ClientOptions#setApplicationId(String)}
      */
+    @Deprecated
     public HttpLogOptions setApplicationId(final String applicationId) {
```

Comment:
Have you reproduced this logic for application ID length, spaces, etc in the new location?

Reply:
Missed it, I added it back. Thanks.